RNiTunes/RNiTunes.m

```diff
@@ -44,9 +44,11 @@ @implementation RNiTunes
     for (MPMediaItemCollection *mediaItemCollection in albumsArray) {
         NSDictionary *albumDictionary = [NSMutableDictionary dictionary];
 
-        MPMediaItem *mediaItem    = [mediaItemCollection representativeItem];
-        NSString    *albumTitle   = [mediaItem valueForProperty:MPMediaItemPropertyAlbumTitle];
-        NSString    *albumArtist  = [mediaItem valueForProperty:MPMediaItemPropertyAlbumArtist];
+        MPMediaItem *mediaItem   = [mediaItemCollection representativeItem];
+        NSString    *albumTitle  = [mediaItem valueForProperty:MPMediaItemPropertyAlbumTitle];
+        NSString    *albumArtist = [mediaItem valueForProperty:MPMediaItemPropertyAlbumArtist];
+        NSDate      *releaseDate = [mediaItem valueForProperty:MPMediaItemPropertyReleaseDate];
+        NSString    *releaseYear = releaseDate == nil ? @"" : [self getYearStringFromDate:releaseDate];
 
```

Comment:
The new attributes are `releaseDate` and `releaseYear`. The other changes are formatting.